brands/amenity/bank.json

```diff
@@ -2534,6 +2534,8 @@
     "tags": {
       "amenity": "bank",
       "brand": "First Bank",
+      "brand:wikidata": "Q5452217",
+      "brand:wikipedia": "en:FirstBank Holding Co",
```

Comment:
This should probably be split up into two different entries. [Overpass Turbo](https://overpass-turbo.eu/s/Hf7) shows that this exists in the USA, Nigeria and even some in Romania. You are referencing only the bank in the US, right?

Reply:
@bonbon12 Ping?

Reply:
Yes it was just the bank in the US.

Reply:
Alright. I can also add the other entries after the merge, but there is another problem:
The wikipedia article (https://en.wikipedia.org/wiki/FirstBank_Holding_Co) says that this bank operates only in three western states of the US. But according to Overpass Turbo there are also banks with that name in eastern states and even in Puerto Rico. There is for example this local bank in Michigan: https://www.first-bank.com/
What is the best thing we can do in that case @bhousel?

Reply:
Hmm yeah "First Bank" is common enough that it's sure to have several separate brands that call themselves that.  I'll merge and add a disambiguation test after the name, like "amenity/bank|First Bank~(Colorado)" so that people in other parts of the USA know not to use it.  (Showing logos helps a lot here too).

We already have a few different ShopRite supermarket brands in the US, so this is not that unheard of.